### PR TITLE
Move the welcome modal to an interstitial, add import button

### PR DESCRIPTION
**Pull request by j-f1**
I noticed that the first run flow is suboptimal for people who want to import an existing file from Actual/YNAB. I’ve moved the welcome modal into the management app and set it up to appear when there are no budgets available (which also has the benefit of allowing people to see the modal again!)

I think there’s some weirdness around getting the modal to reappear when deleting a budget file which I want to work out before merging this.

This PR also reorganizes the management app a bit to reduce usage of modals (currently, hitting escape while the budget list is open leaves you with a blank page).

Screenshot_2023-03-18 08 53 54

- [x] Ensure modal consistently appears when needed (no longer a modal!)
- [x] Fix e2e tests

### CODE REVIEW

File: packages/desktop-client/src/components/manager/WelcomeScreen.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+          Currently Actual implements budgeting based on a{' '}
+          <ExternalLink
+            asAnchor
+            style={{ color: colors.p5 }}
+            href="https://actualbudget.github.io/docs/Budgeting/howitworks"
+          >
+            monthly envelope system
+          </ExternalLink>
+          . In the future, we’ll support multiple ways to do budgeting. We’re
+          also working hard on custom reports and a lot more things.
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/manager/WelcomeScreen.js:48]: **💬 suggestion:** ‏I don't think we should promise future features. Look at the reports page: it has had "more reports coming soon" banner for like 1? maybe 2 years now?
```suggestion
          .
```

File: packages/desktop-client/src/components/manager/WelcomeScreen.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        </P>
+        <P
+          style={{
+            fontWeight: 600,
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/manager/WelcomeScreen.js:52]: **🥜 nitpick:** ‏IMO this should not be bolded. The new welcome page already has a lot of different text formatting options. For me as a user that feels overwhelming. I'm not sure what is more important in the text and what is less important.

File: packages/desktop-client/src/components/manager/WelcomeScreen.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+      >
+        <P>
+          Actual is a personal finance tool that focuses on beautiful design and
+          a slick user experience.{' '}
+          <strong>Editing your data should be as fast as possible.</strong> On
+          top of that, we want to provide powerful tools to allow you to do
+          whatever you want with your data.
+        </P>
+        <P>
+          Currently Actual implements budgeting based on a{' '}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/manager/WelcomeScreen.js:39]: **💭 thought:** ‏maybe this entire paragraph should be dropped. WDYT?

**Review by MatissJanis — CHANGES_REQUESTED** [review]
Overall looks good 👏 

Two small bugs:

1. I'm not able to open an existing budget file that is not downloaded locally `TypeError: l is not a function`
2. There's an interesting "flash" of welcome message


https://user-images.githubusercontent.com/886567/226106776-b7640f15-7a1c-44bc-bf27-bccad9d4be2e.mov

File: packages/desktop-client/src/components/manager/WelcomeScreen.js (hunk 1 above)
Comment (j-f1) [reply packages/desktop-client/src/components/manager/WelcomeScreen.js:48]: 3, maybe 4 :)

This was copied from the old modal but I agree it should be removed.

File: packages/desktop-client/src/components/manager/WelcomeScreen.js (hunk 3 above)
Comment (j-f1) [reply packages/desktop-client/src/components/manager/WelcomeScreen.js:39]: Updated and reorganized it a bit, let me know what you think!

**Comment by MatissJanis** [comment]
Copy LGTM :+1:

Just the two bugs listed above remaining.

**Comment by j-f1** [comment]
Fixed!

**Review by MatissJanis — APPROVED** [review]
Nice!

**Comment by Kidglove57** [comment]
I love this implementation - thank you both!

**Pull request merged**